models/points_test.go

```diff
@@ -975,6 +975,65 @@ func TestParsePointUnicodeString(t *testing.T) {
 	)
 }
 
+func TestParsePointNegativeTimestamp(t *testing.T) {
+	test(t, `cpu value=1 -1`,
+		models.NewPoint(
+			"cpu",
+			models.Tags{},
+			models.Fields{
+				"value": 1.0,
+			},
+			time.Unix(0, -1)),
+	)
+}
+
+func TestParsePointMaxTimestamp(t *testing.T) {
+	test(t, `cpu value=1 9223372036854775807`,
+		models.NewPoint(
+			"cpu",
+			models.Tags{},
+			models.Fields{
+				"value": 1.0,
+			},
+			time.Unix(0, int64(1<<63-1))),
+	)
+}
+
+func TestParsePointMinTimestamp(t *testing.T) {
+	test(t, `cpu value=1 -9223372036854775807`,
+		models.NewPoint(
+			"cpu",
+			models.Tags{},
+			models.Fields{
+				"value": 1.0,
+			},
+			time.Unix(0, -int64(1<<63-1))),
+	)
+}
+
+func TestParsePointInvalidTimestamp(t *testing.T) {
+	_, err := models.ParsePointsString("cpu value=1 9223372036854775808")
+	if err == nil {
+		t.Fatalf("ParsePoints failed: %v", err)
+	}
+	_, err = models.ParsePointsString("cpu value=1 -92233720368547758078")
+	if err == nil {
+		t.Fatalf("ParsePoints failed: %v", err)
+	}
+	_, err = models.ParsePointsString("cpu value=1 -")
+	if err == nil {
+		t.Fatalf("ParsePoints failed: %v", err)
+	}
+	_, err = models.ParsePointsString("cpu value=1 -/")
+	if err == nil {
+		t.Fatalf("ParsePoints failed: %v", err)
+	}
+	_, err = models.ParsePointsString("cpu value=1 -1?")
```

Comment:
Maybe add a test of `1-` as well.  Looks like that should pass but good for coverage.